draft/ydb_object_storage_v1.proto

```diff
@@ -0,0 +1,11 @@
+syntax = "proto3";
+
+package Ydb.ObjectStorage.V1;
+option java_package = "com.yandex.ydb.object_storage.v1";
```

Comment:
To compare with other [services](https://github.com/ydb-platform/ydb-api-protos/blob/master/ydb_query_v1.proto#L5) it will be better to use `tech.ydb.proto.object_storage.v1 `

Reply:
Right, fixed